### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Dumnersm580**
cleaned it up a bit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,53 @@
 # FDM Connector
 
-This OctoPrint plugin will make the initial connection to FDM Monster a breeze. In future it will help in setting up a tunnel connection and tracking filament usage.
+The FDM Connector is an OctoPrint plugin that simplifies the initial connection to FDM Monster. The plugin is also expected to facilitate setting up a tunnel connection and tracking filament usage in future versions.
```
Comment (davidzwa) [inline README.md:5]: We need a big warning on the top: this plugin is currently not in use, nor is there a way to connect it to FDM Monster.

File: README.md [hunk 2]
```diff
@@ -1,45 +1,53 @@
...
 
-The plugin will use `server:host` and `server:port` to give FDM Monster a handle to connect back to this OctoPrint. This is often incorrect, if your OctoPrint is behind a proxy, in a VM, UnRaid, a different device, DMZ, in a docker container or in a VPN.
-At this moment this needs to be rectified in FDM Monster. Later we will allow more advanced ways to fix this, but for now we believe this plugin is going to make it much easier already.
+## Conclusion
+FDM Connector is an OctoPrint plugin that helps in setting up a tunnel connection and tracking filament usage. With its current features, such as auto-registration, it allows you to send your OctoPrint connection parameters to FDM Monster safely, making setting up printers a breeze. Additionally, it offers future features like the Filament Pedometer, Http Tunnel, and Single-sign-On for client-to-machine and machine-to-machine authentication, removing the need for more than one set of credentials across the farm. If you have any feature requests, bugs, or ideas, please head over to https://github.com/fdm-monster/fdm-connector/discussions.
```
Comment (davidzwa) [inline README.md]: Single-Sign-On instead of Single-sign-On

**Comment by davidzwa** [comment]
Really like the changes! My comments above should get it mergeable soon.

**Review by Dumnersm580 — COMMENTED** [review]
fixed according to your respons

**Review by davidzwa — APPROVED** [review]

**Pull request merged**